docs/gcp_setup.md

```diff
@@ -0,0 +1,287 @@
+# Google Cloud Platform Setup for a BigQuery Target
+
+## Overview
+
+This page details Google Cloud components required, with recommended minimal privileges, to use GOE in your GCP project.
+
+### Service Account
+
+A service account should be provisioned from the GCP project. This service account can be used by any service that will execute GOE commands, for example it could be attached to a GCE virtual machine.
+
+### Cloud Storage Bucket
+
+A cloud storage bucket is required to stage data before ingesting it into BigQuery. Ensure the bucket is in a location compatible with the target BigQuery dataset.
```

Comment:
We use GCP and GCE above, do we need to use GCS given that this is BigQuery, and therefore GCS, specific? Else, "cloud storage" should be "Cloud Storage".

Reply:
Fixed